### PR TITLE
Add notebook for loading and starting a trained model from hugging face

### PR DESCRIPTION
**Pull request by TattdCodeMonkey**
This adds a notebook for loading a machine learning model from Hugging Face into your Elasticsearch deployment.

### CODE REVIEW

File: notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb [hunk 1]
```diff
@@ -0,0 +1,316 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Loading Models into Elasticsearch from Hugging Face\n",
+    "\n",
+    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/colab-notebooks-examples/integrations/hugging-face/loading-model-from-hugging-face.ipynb\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
```
Comment (TattdCodeMonkey) [inline notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:9]: I'm not 100% sure if this link is correct, so please correct me if its wrong.

**Comment by TattdCodeMonkey** [comment]
@leemthompo feel free to review copy if you have time

@joemcelroy I tested this locally would be nice to also test against a cloud deployment, but it worked for me locally.
There are a few rough edges that it only works the first time through as some of the commands will fail if run again which is not ideal IMO.

**Comment by TattdCodeMonkey** [comment]
Actually just looking at the issue again I need to add some more steps to the notebook to show how to USE the model :). Will do that as well.

File: notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+    "Now we can instantiate the [Elasticsearch python client](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/index.html), providing the cloud id and password in your deployment."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 34,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
```
Comment (joemcelroy) [inline notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:54]: suggestion: remove stdout stream for examples where they don't help the reader. leave in stdout streams that aid understanding like hit responses.

File: notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb (hunk 1 above)
Comment (joemcelroy) [reply notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:9]: Looks correct to me!

**Comment by joemcelroy** [comment]
> @leemthompo feel free to review copy if you have time
> 
> @joemcelroy I tested this locally would be nice to also test against a cloud deployment, but it worked for me locally. There are a few rough edges that it only works the first time through as some of the commands will fail if run again which is not ideal IMO.

Is it because the model is already loaded? Worth adding at the end a teardown script so you can reset it back.

File: notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb (hunk 1 above)
Comment (leemthompo) [inline notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb:9]:
```suggestion
    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/elastic/elasticsearch-labs/blob/main/notebooks/integrations/hugging-face/loading-model-from-hugging-face.ipynb\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
```

we changed directory names very recently 😃

**Comment by TattdCodeMonkey** [comment]
closing in favor of #17 as it's more complete, we can make updates to it once it merges

**Pull request closed without merging**